File: filedata/entity/java/vipCard.java

package entity;
// 
public class Vipcard  extends BaseDomain{
    public Vipcard() {

    }
    
    //
    private String Id;
        
    //
    private String CompanyStyle;
        
    //
    private String CardNumber;
        
    //
    private String CardPass;
        
    //
    private int(10) Bonus;
        
    //
    private bit(1) Status;
        
    //
    private Date CreatTime;
        
    //
    private Date UseTime;
        
    //
    private String UseCompanyStyle;
        
    //
    private String UseUserName;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public String getCardnumber() {
        return Cardnumber;
    }
    public void setCardnumber(String Cardnumber) {
        this.Cardnumber = Cardnumber;
    }
            
    public String getCardpass() {
        return Cardpass;
    }
    public void setCardpass(String Cardpass) {
        this.Cardpass = Cardpass;
    }
            
    public int(10) getBonus() {
        return Bonus;
    }
    public void setBonus(int(10) Bonus) {
        this.Bonus = Bonus;
    }
            
    public bit(1) getStatus() {
        return Status;
    }
    public void setStatus(bit(1) Status) {
        this.Status = Status;
    }
            
    public Date getCreattime() {
        return Creattime;
    }
    public void setCreattime(Date Creattime) {
        this.Creattime = Creattime;
    }
            
    public Date getUsetime() {
        return Usetime;
    }
    public void setUsetime(Date Usetime) {
        this.Usetime = Usetime;
    }
            
    public String getUsecompanystyle() {
        return Usecompanystyle;
    }
    public void setUsecompanystyle(String Usecompanystyle) {
        this.Usecompanystyle = Usecompanystyle;
    }
            
    public String getUseusername() {
        return Useusername;
    }
    public void setUseusername(String Useusername) {
        this.Useusername = Useusername;
    }
             
    @Override
    public String toString() {
        return "vip_card[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",CardNumber=" + CardNumber + ",CardPass=" + CardPass + ",Bonus=" + Bonus + ",Status=" + Status + ",CreatTime=" + CreatTime + ",UseTime=" + UseTime + ",UseCompanyStyle=" + UseCompanyStyle + ",UseUserName=" + UseUserName + "]";
    }
        
    
}
        

File: filedata/entity/c#/Vipcard.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("vip_card")]
    public class Vipcard
    {
        
        /// <summary>
        /// 主键ID
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 充值卡站点
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 卡号
        /// </summary>
        [Column("CardNumber")]
        public string CardNumber { get; set; }
    
    
        /// <summary>
        /// 充值卡密码
        /// </summary>
        [Column("CardPass")]
        public string CardPass { get; set; }
    
    
        /// <summary>
        /// 充值卡积分面额
        /// </summary>
        [Column("Bonus")]
        public int Bonus { get; set; }
    
    
        /// <summary>
        /// 充值卡状态 0=未使用 1=已使用
        /// </summary>
        [Column("Status")]
        public bool Status { get; set; }
    
    
        /// <summary>
        /// 充值卡创建时间
        /// </summary>
        [Column("CreatTime")]
        public DateTime CreatTime { get; set; }
    
    
        /// <summary>
        /// 充值卡使用时间
        /// </summary>
        [Column("UseTime")]
        public DateTime UseTime { get; set; }
    
    
        /// <summary>
        /// 使用者站点
        /// </summary>
        [Column("UseCompanyStyle")]
        public string UseCompanyStyle { get; set; }
    
    
        /// <summary>
        /// 使用者账号
        /// </summary>
        [Column("UseUserName")]
        public string UseUserName { get; set; }
    
    
    }
}
